apps/src/code-studio/components/aiTutor/compilationTutor.tsx

```diff
@@ -42,10 +44,20 @@ const CompilationTutor: React.FunctionComponent = () => {
   const level = useSelector(
     (state: {aiTutor: AITutorState}) => state.aiTutor.level
   );
+  const scriptId = useSelector(
+    (state: {aiTutor: AITutorState}) => state.aiTutor.scriptId
+  );
   const systemPrompt = compilationSystemPrompt;
 
   const handleSend = async (studentCode: string) => {
-    const chatContext = {systemPrompt: systemPrompt, studentCode: studentCode};
+    const chatContext = {
```

Comment:
is this data mostly/all in redux? Would it make sense to move this object creation into `askAITutor`?